c8/connection.py

```diff
@@ -166,7 +166,7 @@ def set_url_prefix(self, new_prefix):
         self._url_prefix = new_prefix
         #return old_prefix, self._url_prefix
 
-    def send_request(self, request):
+    def send_request(self, request, isUserApi=False):
```

Comment:
Allow specifying the prefix instead of the boolean, it might come in handy later for other endpoints that way.